plugins/task/updatenotification/src/Extension/UpdateNotification.php

```diff
@@ -236,18 +196,22 @@ public function onAfterRender()
         // Send the emails to the Super Users
         foreach ($superUsers as $superUser) {
             try {
-                $mailer = new MailTemplate('plg_system_updatenotification.mail', $jLanguage->getTag());
+                $mailer = new MailTemplate('plg_task_updatenotification.mail', $jLanguage->getTag());
                 $mailer->addRecipient($superUser->email);
                 $mailer->addTemplateData($substitutions);
                 $mailer->send();
             } catch (MailDisabledException | phpMailerException $exception) {
                 try {
-                    Log::add($this->getApplication()->getLanguage()->_($exception->getMessage()), Log::WARNING, 'jerror');
+                    $this->logTask($this->text($exception->getMessage()));
```

Comment:
```suggestion
                    $this->logTask($this->getApplication()->getLanguage()->_($exception->getMessage()));
```
text function was reverted #41385

Reply:
@heelc29 I think we could also use `$this->logTask($jLanguage->_($exception->getMessage()));` here as we have a `$jLanguage = $this->getApplication()->getLanguage();` in line 170.